### PR TITLE
Add more metadata to frontend output

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Two small additions following #2191.

As noted on #1117 we need to provide the meta description, and fix the publisher logo to ensure good results on FB/Twitter/Search.

## Relevant Technical Choices

Uses `get_the_excerpt()` which auto-generates an excerpt as long as the manual excerpt field is missing (#490)

## Testing Instructions

See #1117

---

Addresses #1117

### CODE REVIEW

**Comment by spacedmonkey** [comment]
The excrept is not a field we display in the editor currently. And the post_content will be just amp markup. What would the excrept output? 

Should we just add the excrept field in the editor to make it users can edit the short description?

**Comment by swissspidy** [comment]
> The excrept is not a field we display in the editor currently. And the post_content will be just amp markup. What would the excrept output?

All the text found on the page, minus the markup.

So if you add a text element with the text "Hello World", output will be "Hello World".

> Should we just add the excerpt field in the editor to make it users can edit the short description?

Yes. But that's out of scope for this PR.

I already mentioned it in https://xwp.slack.com/archives/C0135V8LYJE/p1591776036232800

This will be handled by #490.

File: includes/Discovery.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Discovery {
 	 * @return void
 	 */
 	public function init() {
-		add_action( 'web_stories_story_head', [ $this, 'print_metadata' ], 1 );
+		add_action( 'web_stories_story_head', [ $this, 'print_metadata' ] );
```
Comment (spacedmonkey) [inline includes/Discovery.php:44]: Isn't this number required for the title to render?

Comment (spacedmonkey) [reply includes/Discovery.php:44]: This was how it was before.

Comment (swissspidy) [reply includes/Discovery.php:44]: There's no reason why the priority would need to be like that. The title is displayed just fine this way.

**Review by spacedmonkey — APPROVED** [review]

**Pull request merged**